example-config.cfg

```diff
@@ -1,5 +0,0 @@
-[LLM]
```

Comment:
please fill out the example-config to reflect your config.cfg. Or, if you are using .env, create an `.example-env`